Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM registry.access.redhat.com/rhscl/ruby-23-rhel7
+
+# sshd
+USER root
+RUN ["bash", "-c", "yum install -y --setopt=tsflags=nodocs openssh-server libicu-devel && \
+     yum clean all && \
+     sshd-keygen && \
+     mkdir /var/run/sshd && \
+     gem install --no-ri --no-rdoc bunny && \
+     groupadd -r git && \
+     useradd -r -s /bin/bash -g git git && \
+     mkdir --parent /home/git"]
+
+# https://gitlab.com/gitlab-org/gitlab-ce/issues/3027
+# https://docs.docker.com/engine/examples/running_ssh_service/
```

Comment:
@david-martin I stumbled across these links while googling something else, it seems to be where this line came from :)